packages/create-toolpad-app/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "create-toolpad-app",
+  "version": "0.0.42-alpha.5",
+  "keywords": [
+    "react",
+    "toolpad",
+    "mui"
+  ],
+  "description": "Create Toolpad apps with one command",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mui/mui-toolpad",
+    "directory": "packages/create-toolpad-app"
+  },
+  "author": "Toolpad Team",
+  "license": "MIT",
+  "bin": {
+    "create-toolpad-app-test": "./dist/index.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "type": "module",
+  "scripts": {
+    "dev13": "tsc -w -p ./tsconfig.json --outDir ./dist/",
+    "prerelease": "rimraf ./dist/",
+    "release": "tsc -p ./tsconfig.json -outDir ./dist/",
+    "build": "yarn release"
+  },
+  "engines": {
+    "node": ">=14.6.0"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "@types/inquirer": "^9.0.3",
+    "chalk": "^5.2.0",
```

Comment:
I think these will need to go in `dependencies` when we don't use `ncc`

Reply:
does `types/inquirer` have to go in `dependencies` as well?

Reply:
Nope, `devDependencies` should be fine